### DISCUSSION TITLE
Cannot add modules to a Meeting/Layout as expected

### DISCUSSION THREAD

**Discussion by AlexEhn** [tellurianinteractive/Tellurian.Trains.ModulesRegistryApp#228] [Q&A] [answered]
Not sure this is a bug or not. I'll explain what I do and what I expect. 

I have created this layout and meeting. 

Meeting:
https://moduleregistry.azurewebsites.net/meetings/111/view
Layout: 
https://moduleregistry.azurewebsites.net/Meetings/111/Layouts/93

My expectation is then that I can add modules to the meeting. 
I expect to be able to register modules without registering participants because on [this page](https://moduleregistry.azurewebsites.net/MeetingParticipants/565/Layouts/93/RegisterModules), I'm able to see and register modules owned by people who are not registered for the meeting. 

**--> But then I wonder why not all compatible modules are available to register.** 

![image](https://github.com/user-attachments/assets/ba51d022-d9ed-42db-a7d1-0b40f746e3be)

However, If I go to [this page](https://moduleregistry.azurewebsites.net/meetings/111/view)

![image](https://github.com/user-attachments/assets/9c3feade-abc3-4378-a3c9-8b8cff7268ac)

And register the module owner to the meeting, I can add the owners modules.

![image](https://github.com/user-attachments/assets/74103000-cb1f-4d26-ae8d-d3315bba76c3)

![image](https://github.com/user-attachments/assets/043ba396-26a3-4d5d-b905-332cbebdd669)

**Comment by fjallemark** [comment, marked as answer]
When registering a person for a meeting, this is the current procedure:
- Register a person to the meeting: days and time of participation.
- Register the person for one or several layouts in that meeting. If only one layout, the person is automatically added til that layout.
- Eventually register *boxes* with modules. Usually you don't bring modules separate, but in boxes. Each module can be assigned a box label, and a box can contain more than one module.
- Some persons permit group members to borrow their modules, and a group administrator can permits members to borrow group owned modules. These modules also shows when you register modules. If the person is member in several groups, all available modules will be available.
